### DISCUSSION TITLE
Blog Posts Inbox Under Development for Release

### DISCUSSION THREAD

**Discussion by jimmy6DOF** [omigroup/media#77] [General]
These are proposed Bog posts for distribution on Web / W3C mailing list and other channels.

Any suggestions and edits are welcome.

**Comment by jimmy6DOF** [comment]
The blog files are available for viewing and contribution here or in the issues section so this is a non-exhaustive snapshot list of what's active at any point in time

**Comment by indiebio** [comment]
I've moved three drafts so far to github issues, and will add future ones on github too; it's more collaborative. Do we have a folder where these drafts can live? 
see https://github.com/omigroup/media/issues

**Comment by jimmy6DOF** [comment]
Here are a few Under Construction:

https://github.com/omigroup/media/issues/81

https://github.com/omigroup/media/issues/80

https://github.com/omigroup/media/issues/78
